mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -1682,7 +1700,12 @@ func expandComposerEnvironmentConfig(v interface{}, d *schema.ResourceData, conf
   	if err != nil {
   		return nil, err
   	}
-  	transformed.ResilienceMode = transformedResilienceMode
+	if transformedResilienceMode == "RESILIENCE_MODE_UNSPECIFIED" {
```

Comment:
Expand goes from user config --> HTTP request. The user will specify `STANDARD_RESILIENCE` so this should be switched

Reply:
Done.